src/subscription_manager/gui/managergui.py

```diff
@@ -523,7 +523,7 @@ def _getting_started_item_clicked(self, widget):
         try:
             # unfortunately, Gtk.show_uri does not work in RHEL 5
             DEVNULL = open(os.devnull, 'w')
-            subprocess.call(["gnome-open", "ghelp:subscription-manager"],
+            subprocess.call(["xdg-open", "ghelp:subscription-manager"],
```

Comment:
Despite rhel6 includes pygtk 2.16 and PyGtk should include shot_uri since version 2.14, then documentation is not opened. Log includes this:

    2017-04-20 15:13:11,189 [WARNING] subscription-manager-gui:2203:MainThread @managergui.py:529 - Unable to open help documentation: 'module' object has no attribute 'show_uri'